data/mods/Dark-Skies-Above/monsters/alien_cyborgs.json

```
@@ -318,8 +318,8 @@
     "armor_bullet": 35,
     "vision_night": 5,
     "special_attacks": [ { "type": "leap", "cooldown": 10, "max_range": 5 }, { "id": "melee_shock" } ],
-    "path_settings": { "max_dist": 5 },
+    "path_settings": { "max_dist": 5, "avoid_traps": true, "avoid_sharp": true }, },
```

```suggestion
    "path_settings": { "max_dist": 5, "avoid_traps": true, "avoid_sharp": true },
```
I fixed that already... evidently didn't update properly; sigh.